kcc-schema.json

```diff
@@ -0,0 +1,76 @@
+{
+   "$schema":"https://json-schema.org/draft/2020-12/schema",
+   "type":"object",
+   "properties":{
+      "credentialSubject":{
+         "type":"object",
+         "properties":{
+            "id":{
+               "type":"string"
```

Comment:
we could add a regex for a did here and for issuer but this is fine for a first pass

Reply:
updated!